ios/RIBs/Classes/LeakDetector/Executor.swift

```diff
@@ -46,7 +46,7 @@ public class Executor {
 
                 // If we did drop frame, we under-count the frame duration, which is fine. It
                 // just means the logic is performed slightly later.
-                let boundedElapsedTime = min(trueElapsedTime, maxFrameDuration)
+                let boundedElapsedTime = min(trueElapsedTime, Double(maxFrameDuration / 1000))
```

Comment:
Hi all, 
@daidongon found bug:
```swift
Double(maxFrameDuration / 1000) // always return 0
```

should be :

```swift
Double(maxFrameDuration) / 1000
```

@ermik When we can expect RxSwift 5 compatibility release?

Reply:
@avitruk Thanks for reviewing!!  I'll fix it.

Reply:
I don't have any powers as to releases, but together we will make it happen.

Reply:
Time to increase RxSwift version :)